R/pkg/inst/tests/testthat/test_context.R

```diff
@@ -156,7 +160,8 @@ test_that("sparkJars sparkPackages as comma-separated strings", {
 })
 
 test_that("spark.lapply should perform simple transforms", {
-  sc <- sparkR.init()
+  sc <- sparkR.sparkContext()
   doubled <- spark.lapply(sc, 1:10, function(x) { 2 * x })
```

Comment:
Again not related to this PR, but we should remove the need to pass in a SparkContext to this function similar to the DataFrame ones.  I'll file a new JIRA for this

cc @thunterdb @mengxr